scripts/coinweight.lua

```diff
@@ -2,15 +2,17 @@
 --	Please see the LICENSE.md file included with this distribution for attribution and copyright information.
 --
 
+-- Used for item name and also finding the item, so use constant so it's never out of sync.
+COINS_INVENTORY_ITEM_NAME = 'Coins'
+
 ---	This function imports the data from the second column of coins used in damned's coins weight extension.
 --	bmos also used this data structure in an early version of Total Encumbrance.
 --	Once imported, the original database nodes are deleted.
 local function upgradeDamnedCoinWeight(nodeCoinSlot)
 	local nCoinAmount = DB.getValue(nodeCoinSlot, 'amount', 0)
 	local nCoinAmount2 = DB.getValue(nodeCoinSlot, 'amountA', 0)
 	if nCoinAmount2 ~= 0 then
-		local nCoinAmount = nCoinAmount + nCoinAmount2
-		DB.setValue(nodeCoinSlot, 'amount', 'number', nCoinAmount)
+		DB.setValue(nodeCoinSlot, 'amount', 'number', nCoinAmount + nCoinAmount2)
```

Comment:
Luacheck was giving me a shadowed variable name warning on nCoinAmount so I just eliminated it.

Reply:
There isn't anything wrong with intentionally shadowed variables, but I agree this is a better solution :)